stories/utils/generate-widget-stories.js

```diff
@@ -203,6 +203,7 @@ export function generateReportBasedWidgetStories( args ) {
 		ZeroData:
 			typeof zeroing === 'function'
 				? {
+						features: [ 'adsenseSetupV2' ],
```

Comment:
Seeing as this is within a generic utility function that's used in a few places, I think it would be preferable, as a matter of principle, not to hard-wire this feature flag into this story.

Instead, I'd suggest adding some `defaultVariantOptions`, for example:

```diff
diff --git a/stories/module-adsense-components.stories.js b/stories/module-adsense-components.stories.js
index 58a05d2427..703d071702 100644
--- a/stories/module-adsense-components.stories.js
+++ b/stories/module-adsense-components.stories.js
@@ -101,6 +101,11 @@ generateReportBasedWidgetStories( {
 	datastore: MODULES_ADSENSE,
 	group: 'AdSense Module/Components/Module/Overview Widget',
 	referenceDate: '2020-11-25',
+	defaultVariantOptions: {
+		ZeroData: {
+			features: [ 'adsenseSetupV2' ],
+		},
+	},
 	zeroing,
 	...generateAdSenseData( [
 		{
diff --git a/stories/utils/generate-widget-stories.js b/stories/utils/generate-widget-stories.js
index 63354ee6cd..34712d8fbe 100644
--- a/stories/utils/generate-widget-stories.js
+++ b/stories/utils/generate-widget-stories.js
@@ -65,6 +65,7 @@ export function generateReportBasedWidgetStories( args ) {
 		Component,
 		referenceDate,
 		wrapWidget = true,
+		defaultVariantOptions = {},
 		additionalVariants = {},
 		additionalVariantCallbacks = {},
 		setup = () => {},
@@ -134,6 +135,7 @@ export function generateReportBasedWidgetStories( args ) {
 	// Existing default variants.
 	const defaultVariants = {
 		Loaded: {
+			...defaultVariantOptions.Loaded,
 			callback( { dispatch } ) {
 				if ( Array.isArray( options ) ) {
 					options.forEach( ( option, index ) => {
@@ -152,6 +154,7 @@ export function generateReportBasedWidgetStories( args ) {
 			},
 		},
 		Loading: {
+			...defaultVariantOptions.Loading,
 			callback( { dispatch } ) {
 				if ( Array.isArray( options ) ) {
 					options.forEach( ( option, index ) => {
@@ -176,6 +179,7 @@ export function generateReportBasedWidgetStories( args ) {
 			},
 		},
 		DataUnavailable: {
+			...defaultVariantOptions.DataUnavailable,
 			callback( { dispatch } ) {
 				if ( Array.isArray( options ) ) {
 					options.forEach( ( option, index ) => {
@@ -203,7 +207,7 @@ export function generateReportBasedWidgetStories( args ) {
 		ZeroData:
 			typeof zeroing === 'function'
 				? {
-						features: [ 'adsenseSetupV2' ],
+						...defaultVariantOptions.ZeroData,
 						callback: ( { dispatch } ) => {
 							if ( Array.isArray( options ) ) {
 								options.forEach( ( option, index ) => {
@@ -235,6 +239,7 @@ export function generateReportBasedWidgetStories( args ) {
 				  }
 				: undefined,
 		Error: {
+			...defaultVariantOptions.Error,
 			callback( { dispatch } ) {
 				const error = {
 					code: 'missing_required_param',
```